src/monaco-utils.ts

```diff
@@ -53,4 +53,22 @@ export class MonacoUtils {
     MonacoUtils.TreeDefaults = TreeDefaults;
   }
 
+  static expandTree(tree: any) {
```

Comment:
I'd change this to something more specific like `expandDirectories(tree: any, directories: string[])`

Reply:
still remain the arg but change name, it is not good to take that const out of function because it will not used again for the directorytree class

Reply:
> it is not good to take that const out of function because it will not used again for the directorytree class

Ideally the const or even `expandTree` call itself shouldn't even be in directory tree class, as it is generic UI component. So which directories to expand better be provided from outside. It's ok to keep as is for now, but I then would at least rename to `expandCodeDirectories" or something like that (as it doesn't expand whole tree)